assets/js/planning.js

```diff
@@ -43,13 +43,28 @@ function initDatesRange($picker, $from, $to, withTime) {
     $to.val(picker.endDate.format('YYYY-MM-DDTHH:mm'));
   });
 
+  $picker.on('showCalendar.daterangepicker', function () {
+    hourselectOnlyEvenHour();
+  });
+
   $picker.on('cancel.daterangepicker', function () {
     $picker.val('');
     $from.val('');
     $to.val('');
   });
 }
 
+function hourselectOnlyEvenHour() {
+  let allSelectHours = document.getElementsByClassName('hourselect');
```

Comment:
you should use a jQuery selector and look for this item only in the current datepicker

Reply:
e.g.:

```js
function hourselectOnlyEvenHour($picker) {
    const allSelectHours = $picker.find('.hourselect');
}
```